python/ray/tests/test_client_init.py

```diff
@@ -167,23 +167,23 @@ def mock_connection_response():
     server_handle.data_servicer._build_connection_response = \
         mock_connection_response
 
-    ray = RayAPIStub()
+    ray = _ClientContext()
     with pytest.raises(RuntimeError):
-        _ = ray.connect("localhost:50051")
+        _ = ray._connect("localhost:50051")
 
-    ray = RayAPIStub()
-    info3 = ray.connect("localhost:50051", ignore_version=True)
+    ray = _ClientContext()
+    info3 = ray._connect("localhost:50051", ignore_version=True)
     assert info3["num_clients"] == 1, info3
-    ray.disconnect()
+    ray._disconnect()
 
 
 def test_protocol_version(init_and_serve):
     server_handle = init_and_serve
-    ray = RayAPIStub()
-    info1 = ray.connect("localhost:50051")
+    ray = _ClientContext()
+    info1 = ray._connect("localhost:50051")
     local_py_version = ".".join([str(x) for x in list(sys.version_info)[:3]])
     assert info1["protocol_version"] == CURRENT_PROTOCOL_VERSION, info1
-    ray.disconnect()
+    ray._disconnect()
```

Comment:
Why these changes to _disconnect()?

Reply:
I previously thought there might be a conflict. Maybe it's not the right fix. I'll follow it up and either give you some explanation or revert it back.